src/pages/studio/Studio.module.css

```diff
@@ -0,0 +1,120 @@
+.page {
+  min-height: 100vh;
+  width: 100%;
+}
+
+.blurb {
+  @mixin body-3;
+  width: 100%;
+  margin-bottom: 80px;
+
+  @media (--tablet) {
+    @mixin body-1;
+  }
+
+  @media (--desktop) {
+    max-width: 1020px;
+    margin-bottom: 112px;
+  }
+}
+
+.skills {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  column-gap: 20px;
+  margin-bottom: 80px;
+
+  @media (--tablet) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (--desktop) {
+    margin-bottom: 172px;
+  }
+
+  & .list {
+    margin-bottom: 36px;
+  }
+
+  & .heading {
+    color: var(--link-color);
+    margin-bottom: 16px;
+  }
+
+  & :global(li):not(.heading) {
+    margin-bottom: 6px;
+
+    @media (--desktop) {
+      margin-bottom: 12px;
+    }
+  }
+}
+
+
+.carousel,
+.block {
+  width: 100vw;
+  margin-left: -20px;
+  margin-bottom: 80px;
+
+  @media (--tablet) {
+    margin-left: -40px;
+  }
+
+  @media (--desktop) {
+    width: 100%;
+    margin-left: 0;
+    margin-bottom: 188px;
+  }
+}
+
+.carousel {
+  padding-left: 20px;
+
+  @media (--desktop) {
+    padding-left: 0;
+    margin-left: calc((100vw - var(--wrapper-width)) / 2);
+  }
+
+  @media (--hd) {
+    padding-left: 0;
+    margin-left: calc((1920px - var(--wrapper-width)) / 2);
+  }
```

Comment:
Seeing them next to each other, questioning the difference between the two mqs

Reply:
Revisited, `padding-left: 0;` doesn't have to be put in `--hd` as the `--desktop` styling also applies to `--hd`